src/Umbraco.Cms.Persistence.EFCore/Factories/UmbracoPooledDbContextFactory.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Persistence.EFCore.Factories;
+
+/// <inheritdoc/>
+internal class UmbracoPooledDbContextFactory<TContext> : PooledDbContextFactory<TContext>
```

Comment:
How can this be a new file. It already exists in v12/dev

Reply:
It is the same as the one on `v12/dev` cherry picked from your commit but this PR targets `contrib` where the commit hasn't been merged yet. I cherry picked it to make sure it worked with the Factory.